pkg/util/oom/oom_linux.go

```diff
@@ -78,12 +75,22 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 	value := strconv.Itoa(oomScoreAdj)
 	var err error
 	for i := 0; i < maxTries; i++ {
-		if err = ioutil.WriteFile(oomScoreAdjPath, []byte(value), 0700); err != nil {
+		f, err := os.Open(oomScoreAdjPath)
+		if err != nil {
+			if syscallNotExists(err) {
+				return os.ErrNotExist
```

Comment:
Perhaps in a follow-up, I'd recommend just returning `err` and modifying `pkg/kubelet/dockertools/manager.go` to call `os.IsNotExist(err)` instead of checking for `os.ErrNotExist`.